### PR TITLE
Staffbase Connector

### PR DESCRIPTION
**Pull request by Nm-jens**
---
Please check the following conditions for your PR.

- [x] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [x] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

**Comment by ptalrejagithub** [comment]
@Nm-jens, can you please sign the CLA.

**Comment by ptalrejagithub** [comment]
@Nm-jens Can you please add the readme.md too.

File: custom-connectors/Staffbase/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2691 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Staffbase",
+    "description": "This connector provides access to your Staffbase instance",
```
Comment (sriyen-msft) [inline custom-connectors/Staffbase/apiDefinition.swagger.json:5]: @Nm-jens Can you please provide a detailed description value which any one can understand.

Documentation - [Description](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#description)

File: custom-connectors/Staffbase/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2691 @@
...
+    "version": "1.0",
+    "contact": {
+      "name": "Staffbase Support",
+      "url": "https://www.staffbase.com",
+      "email": "support@staffbase.com"
+    }
+  },
+  "host": "powerautomateproxy-main-de1.staffbase.com",
```
Comment (sriyen-msft) [inline custom-connectors/Staffbase/apiDefinition.swagger.json:13]: @Nm-jens Please make sure that the host URL is a production version of your service/API

File: custom-connectors/Staffbase/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2691 @@
...
+                              }
+                            },
+                            "description": "localization"
+                          }
+                        },
+                        "description": "config"
+                      },
+                      "spaceID": {
+                        "type": "string",
+                        "description": "spaceID"
```
Comment (sriyen-msft) [inline custom-connectors/Staffbase/apiDefinition.swagger.json:78]: @Nm-jens Please provide valid description values for the properties as these are looking like IDs

Documentation - [Description values](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the invalid description values in the swagger

File: custom-connectors/Staffbase/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API-Schl\u00fcssel",
```
Comment (sriyen-msft) [inline custom-connectors/Staffbase/apiProperties.json:7]: @Nm-jens Please replace the unicodes with English language characters in all the connector artifacts as they would not be supported in all the Power Automate pages.

File: custom-connectors/Staffbase/apiProperties.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Specify your hosting url: hostingURL",
+          "displayName": "Hosting URL",
+          "tooltip": "Provide hosting URL here"
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```
Comment (sriyen-msft) [inline custom-connectors/Staffbase/apiProperties.json:29]: @Nm-jens Please make sure that the given iconBrandColor is the background color of your connector's icon.

Documentation - [Icon](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#icon)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
> I will check these issues today.

@Nm-jens I'm just checking in to see how things are going with updating your PR to address the feedback. Feel free to reach out with any questions you may have.

**Comment by sriyen-msft** [comment]
> I will check these issues today.

@Nm-jens I'm just checking in to see how things are going with updating your PR to address the feedback. Feel free to reach out with any questions you may have.

**Review by mamurshe — CHANGES_REQUESTED** [review]
If this is a certified connector please move to certified-connectors folder.

**Comment by sriyen-msft** [comment]
@Nm-jens I'm just checking in to see how things are going with updating your PR to address the feedback. Feel free to reach out with any questions you may have.

**Comment by sriyen-msft** [comment]
@Nm-jens I'm just checking in to see how things are going with updating your PR to address the feedback. Feel free to reach out with any questions you may have.

**Comment by sriyen-msft** [comment]
@Nm-jens I am closing this PR since there is no response from you, you can re-open the PR whenever you are ready for certification. Thanks

**Pull request closed without merging**